packages/theme/pages/Product.vue

```diff
@@ -214,8 +214,11 @@ export default {
       }
     ]);
 
-    // TODO: Implement breadcrumbs
-    // const breadcrumbs = computed(() => productGetters.getBreadcrumbs ? productGetters.getBreadcrumbs(product.value) : props.fallbackBreadcrumbs);
+    const breadcrumbs = computed(() =>
+      productGetters.getBreadcrumbs
```

Comment:
I don't think this check is appriopriate. `getBreadcrumbs` is a getter function so it should not be used as a condition. Are you using fallbackBreadcrumbs anyway?

Reply:
I only uncommented this line. I see what you mean. There's no `fallbackBreadcrumbs` anymore as you say.